server_manager/electron_app/release/macos.entitlements

```diff
@@ -4,6 +4,8 @@
   <dict>
     <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
     <true/>
+    <key>com.apple.security.cs.allow-jit</key>
```

Comment:
Is this really necessary?

This says it "may" be needed, but doesn't explain when:
https://github.com/electron/notarize?tab=readme-ov-file#prerequisites

Have you tried without it?